dashboard/pom.xml

```diff
@@ -67,6 +67,37 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                      <id>add-resource</id>
+                      <phase>generate-resources</phase>
+                      <goals>
+                        <goal>add-resource</goal>
```

Comment:
AFAIK here you're adding sources as resources 
while we should have a separated artifact including the sources.
Else we will deploy a war file that will include the source code while we're only requiring final js code

so IMHO it's more about add-source goal and using maven-source-plugin